modules/ROOT/pages/cloudhub-deployment-settings.adoc

```diff
@@ -0,0 +1,34 @@
+= Configuring CloudHub Deployment Settings
```

Comment:
Is this page replacing cloudhub-deploy-options? If so, add a page alias here to redirect.